example/default.yaml

```diff
@@ -30,14 +30,14 @@ keybinds:
           key: [ Alt: ']',]
     locked:
         - action: [SwitchToMode: Normal,]
-          key: [Ctrl: 'g',]
+          key: [Ctrl: 'c',]
```

Comment:
I don't think changing the mode per default is a good idea for now. 
At the moment we settled on g for the locked mode. Please do feel free to open a separate issue for a proposed change, so we can discuss it.

Reply:
This here is _the_ issue though to simulate tmux keybinds, I cannot find a way to have there exist a tmux like prefix keybind that is then followed by the needed tmux-like keys at least as far as I can tell in this yaml layout. I attempted to create a new mode named tmux with that keybind, but the yaml doesn't accept it as a new mode is not listed or not in the keybinds shown at the bottom of the screen, listing this error when loading the yaml:

`Deserialisation error: keybinds: unknown variant tmux, expected one of Normal, Locked, Resize, Pane, Tab, Scroll, RenameTab at line 3 column 11`

I have two alternative ideas for work-arounds, I'll discuss in the body bellow, but skimming that, I think you see the issue here already.

I honestly didn't care for this workaround either, I thought it looked messy, but was the most friendly currently to a tmux user as they don't have a lock mode, though it does ruin that feature in the process which isn't so good. :(

Reply:
Please change this back to g, this is the default file. This should not have the tmux compatible keybinds.